tests/test_D1_model_components.py

```diff
@@ -718,6 +730,145 @@ def test_storage_fix(self):
         assert self.model.entities[-1].invest_relation_input_capacity is None
         assert self.model.entities[-1].invest_relation_output_capacity is None
 
+        assert (
+            self.model.entities[-1].fixed_losses_relative.default
+            == dict_asset[STORAGE_CAPACITY][THERM_LOSSES_REL][VALUE]
+        )
+        assert (
+            self.model.entities[-1].fixed_losses_absolute.default
+            == dict_asset[STORAGE_CAPACITY][THERM_LOSSES_ABS][VALUE]
+        )
+
+    def test_storage_optimize_investment_minimum_0_float(self):
+        # Test if minimum value is zero if thermal losses are zero (float)
+        dict_asset = self.dict_values[ENERGY_STORAGE]["storage_optimize"]
+        dict_asset[STORAGE_CAPACITY][MAXIMUM_CAP] = {VALUE: None, UNIT: "kWh"}
+        dict_asset[INPUT_POWER][MAXIMUM_CAP] = {VALUE: None, UNIT: "kWh"}
+        dict_asset[OUTPUT_POWER][MAXIMUM_CAP] = {VALUE: None, UNIT: "kWh"}
```

Comment:
If you add `maximumCap` to the `tests/inputs/mvs_config.json` you don't have to add them in each test:
```
"maximumCap": {
                "unit": "None or kWh",
                "value": None
            },
```
For the losses, however, I agree that it makes sense to add them here, as they are different for each test.

Reply:
Since I orientated my tests on the already existing tests in `TestStorageComponent` class, I have concerns changing that. It has been implemented this way in test_storage_optimize().

What do you think @smartie2076 ?

Reply:
I wrote these tests and the json file before `maximumCap` existed, which is why it is not included. So, especially because you set it to `None` it's fine if you add it.

Reply:
> I wrote these tests and the json file before `maximumCap` existed, which is why it is not included. So, especially because you set it to `None` it's fine if you add it.

Cool! :)
Done with commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/0eab74fff754e0e6f1adce62e6545c185cc2fa27